### DISCUSSION TITLE
CommandDotNet 7, Dotnet 6.0

### DISCUSSION THREAD

**Discussion by atari-monk** [bilal-fazlani/commanddotnet#444] [Q&A] [answered]
Hi @drewburlingame 
Hope you dont mind me asking this.
Is version 7 with REPL ?

**Comment by drewburlingame** [comment, marked as answer]
I don't mind at all.  REPL is still just a branch.  There are a lot of tests to write to prevent regressions and I just don't have the time to do it atm.

**Reply by atari-monk** [reply]
I keep having thouth "one of these days i will actually read this lib code to increase wisdom"
